KsApi/mutations/inputs/CreateApplePayBackingInputTests.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+@testable import KsApi
+import XCTest
+
+final class CreateApplePayBackingInputTests: XCTestCase {
+  func testCreateApplePayBacking_toInputDictionary_noNilValues() {
+    let input = CreateApplePayBackingInput(
+      amount: "10.00",
+      locationId: 123,
+      paymentInstrumentName: "instrumentName",
+      paymentNetwork: "paymentNetwork",
+      projectId: 12_345,
+      rewardId: 321,
+      stripeToken: "stripeTokenXYZ",
+      transactionIdentifier: "transactionId"
+    )
+
+    let inputDictionary = input.toInputDictionary()
+
+    XCTAssertEqual(inputDictionary["amount"] as! String, "10.00")
```

Comment:
I know that these are unit tests but I would be more careful force-unwrapping things like this...because if something happens the test would not fail, but rather the test would crash. This might be during deployment, etc. so being a little bit more safe and intentional we can use the following approach (which should fail the test if conditions change, but not crash it).

IMO failing test is usually better more understandable than crash in console reported by CircleCI.

```swift
XCTAssertEqual(inputDictionary["amount"] as? String, "10.00")
...
```

What do you think?